exchange-common/src/main/scala/com/vesoft/exchange/common/processor/Processor.scala

```diff
@@ -230,4 +231,19 @@ trait Processor extends Serializable {
     else assert(assertion = false, context)
   }
 
+  def customRepartition(spark: SparkSession,
+                        data: Dataset[(Array[Byte], Array[Byte])],
+                        partitionNum: Int): Dataset[(Array[Byte], Array[Byte])] = {
+    import spark.implicits._
+    data.rdd
```

Comment:
why don't use `repartition` directly?

Reply:
> why don't use `repartition` directly?

Dataframe doesn't have `repartition` function, it's RDD's function.